### DISCUSSION TITLE
"we do not understand how quickly the Universe is expanding"

### DISCUSSION THREAD

**Discussion by RedshiftDrift** [a-cosmology-group/acg#152] [Cosmology Models]
"One of the biggest problems facing modern astronomers, astrophysicists, and cosmologists — every scientist who studies the Universe itself for a living — is that [we do not understand how quickly the Universe is expanding](https://bigthink.com/starts-with-a-bang/triple-lens-supernova-jwst/)"
Big Think, Sept. 20, 2023.

Since the universe is not expanding, Ethan is right on this one!

**Comment by budrap00** [comment]
We also do not understand how many angels can dance on the head of a pin. Fantasy physics is hard!

**Reply by bgreenman** [reply]
Which is right, physics trapped inself in a failed model
